dev/docker_data/default_pipeline.yml

```diff
@@ -80,6 +80,13 @@ pipeline_setup:
 
   system_config:
 
+    # Random seed used to fix the state of execution.
+    # If unset, each process uses its own default.
+    # If set, a `random.log` file will be generated logging the random state used by each process.
+    # If set to a positive integer (up to 2147483647), that integer will be used to seed each process.
+    # If set to 'random', a random seed will be generated and recorded for each process.
```

Comment:
In this case, if I were to regenerate derivatives in an empty directory aside from this log file, would the new run respect the random seeds for each task? It's easy in the positive integer case, I just provide the same value, but I wanted to be sure about the re-executability here.

Reply:
Right now the log is just a log and doesn't influence future runs. But I think we should update so the generated `cpac_pipeline_config_{timestamp}_min.yml` for a `random` run logs the generated seed instead of 'random'

Reply:
So, if I understand correctly, we cannot replicate a run used with `random`? Indeed, I think it's important to add that functionality if we're providing this option, otherwise in practice it is not much different from `None`.

Reply:
My thinking before was if you wanted to replicate, you'd just look at the log from the initial `--random_seed random` run, see what the seed was set at there, and manually provide that value for the subesquent run. But 792b3ffcabf97de1c10754f0978fb2c5437c7b12 makes the change so the `cpac_pipeline_config_{timestamp}.yml` / `cpac_pipeline_config_{timestamp}_min.yml` files generated at runtime can be used as pipeline configs for replication runs.

Reply:
Awesome, thanks so much! I think this increases clarity on how to use the feature for non c-pac dev